plugins/@grouparoo/salesforce/src/lib/export/mapping.ts

```diff
@@ -86,22 +86,22 @@ const supportedTypeMap = {
   id: "string",
   picklist: "string",
   currency: "float",
+  masterrecord: "string", // When records are merged, the ID of the record that is saved
+  reference: "string", // Cross-references to a different object. Analogous to a foreign key field in SQL.
+  base64: "string", // Base 64-encoded binary data.
+  date: "string",
   //TODO:
-  address: null,
-  date: null,
+  address: null, //A compound data type that contains address field data.
```

Comment:
Is there anything we can do to export this better?  For example, what if was a grouparoo string, and upon export we sent something like `{street_address: ${value}}` - would that work?

Reply:
hmmm, good idea. I'll try this out.

Reply:
It looks like there is no way to create a new Address field, and in contacts, we don't receive it directly we receive it spread like this:  
`{...
MailingStreet: null,
      MailingCity: null,
      MailingState: null,
      MailingPostalCode: null,
      MailingCountry: null,
      MailingLatitude: null,
      MailingLongitude: null,
      MailingGeocodeAccuracy: null,
      MailingAddress: null,
      Phone: null,
      Fax: null,
      MobilePhone: null,
      ...
}`

Reply:
Dang. Ok then, thanks for checking!